openmc/mgxs/mgxs.py

```diff
@@ -7,13 +7,14 @@
 import sys
 import copy
 import abc
+import itertools
 
 import numpy as np
 
 import openmc
 import openmc.checkvalue as cv
 from openmc.mgxs import EnergyGroups
-
+from openmc import Mesh
```

Comment:
Nope, and Mesh is never actually used like this. Must have needed it when somewhere in the development process. I'll take this out.